File: ruby/control_flow/if_statement/if_else_multiple_statement.rb
# Copyright (C) 2019-2026 Katarzyna Krasińska
# Languagium - https://github.com/katheroine/languagium
# Licensed under GPL-3.0 - see LICENSE.md

if 1 > 2
  puts '1 > 2'
elsif 2 > 3
  puts '2 > 3'
else
  puts '!(1 > 2) && !(2 > 3)'
end

if 1 > 2
  puts '1 > 2'
elsif 1 > 0
  puts '1 > 0'
else
  puts '!(1 > 2) && !(1 > 0)'
end

if 2 > 1
  puts '2 > 1'
elsif 1 > 0
  puts '1 > 0'
else
  puts '!(2 > 1) && !(1 > 0)'
end

File: javascript/control_flow/if_statement/if_else_multiple_statement.js
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

if (1 > 2)
  console.log('1 > 2');
else if (2 > 3)
  console.log('2 > 3');
else
  console.log('!(1 > 2) && !(2 > 3)');

if (1 > 2)
  console.log('1 > 2');
else if (1 > 0)
  console.log('1 > 0');
else
  console.log('!(1 > 2) && !(1 > 0)');

if (2 > 1)
  console.log('2 > 1');
else if (1 > 0)
  console.log('1 > 0');
else
  console.log('!(2 > 1) && !(1 > 0)');
